src/main/java/seedu/address/model/person/Score.java

```diff
@@ -0,0 +1,63 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's score in JABPro.
+ */
+public class Score implements Comparable<Score> {
+    public static final String MESSAGE_CONSTRAINTS = "Score should be a non-negative integer";
+    public final int value;
+
+
+    /**
+     * Constructs a {@code Score}.
+     * @param value
+     */
+    public Score(int value) {
+        requireNonNull(value);
+        checkArgument(isValidScore(value), MESSAGE_CONSTRAINTS);
+        this.value = value;
+    }
+
+    /**
+     * Returns if a given integer is a valid score.
+     * @param test integer to be tested
+     * @return true if test is a non-negative integer
+     */
+    public static boolean isValidScore(int test) {
+        return test >= 0;
```

Comment:
As above, perhaps we could enforce a constraint on what the max score is?

Reply:
Addressed in the other comment